src/main/java/ch/njol/skript/expressions/ExprCoordinate.java

```diff
@@ -38,14 +39,17 @@
  * @author Peter Güttinger
  */
 @Name("Coordinate")
-@Description("Represents a given coordinate of a location. ")
+@Description("Represents a given coordinate of a location or a chunk. The X/Z coordinates of a chunk will differ from a location in the sense they " +
+	"are basically a floored version of (x coord of location / 16). The Y coordinate is non existent for a chunk so this will always return 0. " +
+	"Coordinates of a location can be changed, but can not be changed for a chunk.")
 @Examples({"player's y-coordinate is smaller than 40:",
-		"	message \"Watch out for lava!\""})
-@Since("1.4.3")
-public class ExprCoordinate extends SimplePropertyExpression<Location, Number> {
+		"	message \"Watch out for lava!\"",
+		"set {_x} to x coord of chunk at player"})
+@Since("1.4.3, INSERT VERSION (coords of chunk)")
+public class ExprCoordinate extends SimplePropertyExpression<Object, Number> {
 	
 	static {
-		register(ExprCoordinate.class, Number.class, "(0¦x|1¦y|2¦z)(-| )(coord[inate]|pos[ition]|loc[ation])[s]", "locations");
+		register(ExprCoordinate.class, Number.class, "(0¦x|1¦y|2¦z)(-| )(coord[inate]|pos[ition]|loc[ation])[s]", "locations/chunks");

```

Comment:
We could maybe send an error when users try to get the y coordinate from a chunk, of course, this will only work when the type is known but that's probably better than nothing.

It's either that or maybe the users might find it returning the world's max height useful? It's worth discussing.